configs/dev-config.yaml

```diff
@@ -34,10 +34,6 @@ default_data:
           ],
         "currPosition": 0,
       },
-      {
```

Comment:
This was needed to deploy or just cutting down on extra lines?

Reply:
Wasn’t technically needed for the deploy but it’s a bug that I wanted to fix! Because based on our discussion yesterday, a scenario with 0 scenes should have 0 transitions (so only the intro). And since we use the default for scenario creation, it makes sense to fix it

Reply:
Ohh right that makes sense